decred/build.py

```diff
@@ -0,0 +1,18 @@
+"""
+Copyright (c) 2020, The Decred developers
+See LICENSE for details
+"""
+
+try:
+    from Cython.Build import cythonize
+except ImportError:
```

Comment:
When loading `decred` as a dependency for `tinywallet`, Github has problems with this import. Disabling it just for CI should not be a problem.